package.json

```diff
@@ -58,7 +58,7 @@
     "protractor": "~5.1.2",
     "request-promise": "^4.2.1",
     "ts-node": "~3.2.0",
-    "tslint": "~5.3.2",
+    "tslint": "^5.3.2",
```

Comment:
why tslint is aded in package.json, its already been added you dont have to change it

Reply:
@razu9861 I accidentally changed it form "~" to "^"

Reply:
Yes, this change is not required for the given issue. The issue requires only rating function . So, you don;t have to change it. Could you revert it back ?

Reply:
@nirojdyola Reverted